helm/cluster-api/files/bootstrap/patches/versions/v1alpha3/kubeadmconfigtemplates.bootstrap.cluster.x-k8s.io.yaml

```diff
@@ -0,0 +1,652 @@
+- op: replace
```

Comment:
Kustomize patch with v1alpha3 OPEN API schema for `kubeadmconfigtemplates.bootstrap.cluster.x-k8s.io`.